File: 백준/Bronze/1934. 최소공배수/최소공배수.py
T = int(input())
def GCD(A,B):
    r = A%B
    if r == 0:
        return B
    return GCD(B,r)
for i in range(T):
    A, B = map(int,input().split())
    c = GCD(A,B)
    print(A*B//c)

File: 백준/Silver/13241. 최소공배수/최소공배수.js
const input = require('fs').readFileSync('dev/stdin').toString().trim().split('\n');

const arr = input[0].split(' ').map(Number);

let max = Math.max(...arr);
let min = Math.min(...arr);

function GCD(A, B) {
	if (B === 0) return A;

	let max = Math.max(A, B);
	let min = Math.min(A, B);

	return GCD(min, max % min);
}

console.log((max * min) / GCD(max, min));
